configs/ray-summit-stable-diffusion/aws.yaml

```diff
@@ -0,0 +1,17 @@
+cloud: education-us-west-2
+head_node:
+  instance_type: g5.4xlarge
+  resources:
+    CPU: 16
+    GPU: 1
+  flags: {}
+worker_nodes:
+  - instance_type: g5.4xlarge
+    flags: {}
+    name: '1xA10G:16CPU-64GB'
+    min_nodes: 1
+    max_nodes: 1
+    market_type: ON_DEMAND
+enable_cross_zone_scaling: false
+flags:
+  allow-cross-zone-autoscaling: false
```

Comment:
Only the `enable_cross_zone_scaling` needs to be present here, and I would set it to `true`

Reply:
This is the output of "export yaml" 

So for context, we intend to perform an on-demand capacity reservation in a specific AZ and we want to ensure we make use of these resources. 

Are you suggesting we keep it to true, just in case as a backup if capacity is reached ?

Reply:
@shomilj given the context I provided about ODCR do you still recommend setting `enable_cross_zone_scaling` to true ?

Reply:
Ok if you have OCDR then you can probably disable. But you may need to either ask users to hard-code a zone...since I don't think you can fix the zone in this compute config, so the SIM will pick any zone (not necessarily the one with your OCDR).